libqtile/resources/default_config.py

```diff
@@ -138,6 +138,8 @@
                 ),
                 widget.TextBox("default config", name="default"),
                 widget.TextBox("Press &lt;M-r&gt; to spawn", foreground="#d75f5f"),
+                # NB Systray is incompatible with Wayland, consider using StatusNotifier instead
+                # widget.StatusNotifier(),
                 widget.Systray(),
```

Comment:
Should we add an if block above this block with something like this?:
```py
from libqtile import qtile  # at the top

if qtile.core.name == "x11":
    tray = widget.Systray()
elif qtile.core.name == "wayland":
    tray = widget.StatusNotifier()
```

What do you think? 

and then just `tray` here, obviously.

Reply:
Oh yes.................... ignore me :sweat_smile: